include/kokkos-utils/callbacks/TimerListener.hpp

```diff
@@ -0,0 +1,109 @@
+#ifndef KOKKOS_UTILS_CALLBACKS_TIMERLISTENER_HPP
+#define KOKKOS_UTILS_CALLBACKS_TIMERLISTENER_HPP
+
+#include "Kokkos_NumericTraits.hpp"
+
+#include "kokkos-utils/callbacks/Listener.hpp"
+#include "kokkos-utils/callbacks/Matcher.hpp"
+#include "kokkos-utils/concepts/ExecutionSpace.hpp"
+
+namespace Kokkos::utils::callbacks
+{
+
+template <typename TimerType>
+concept StartableWithoutEvent = requires (TimerType& timer) {
```

Comment:
We could name the "no argument" version simply as `Startable` and `Stoppable`.

Reply:
OK. Better indeed!